### PR TITLE
(docs): update drag-outside-window of dialog-box

### PR DESCRIPTION
**Pull request by James-9696**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [* ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [* ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [* ] No

### CODE REVIEW

**Review by AcWrong02 — CHANGES_REQUESTED** [review]
默认值是false的话，这边使用true来进行演示是不是更合理一点呢？
另外，选项式api对应的文件draggable.vue对应的代码也需要一起修改一下

**Comment by James-9696** [comment]
已修改review建议

File: examples/sites/demos/pc/app/dialog-box/webdoc/dialog-box.js [hunk 1]
```diff
@@ -159,9 +159,9 @@ export default {
       'name': { 'zh-CN': '可拖拽的弹窗', 'en-US': 'Dragable pop-up window' },
       'desc': {
         'zh-CN':
-          '<p>可通过<code>draggable</code>属性设置<code>true</code>，鼠标点击标题区域拖拽。具体事件：<code>@drag-start</code><code>@drag-move</code><code>@drag-end</code>。</p>\n',
+          '<p>可通过<code>draggable</code>属性设置<code>true</code>，鼠标点击标题区域拖拽；通过<code>drag-outside-window</code>属性设置<code>true</code>，将弹窗拖出窗口。具体事件：<code>@drag-start</code><code>@drag-move</code><code>@drag-end</code>。</p>\n',
```
Comment (gimmyhehe) [inline examples/sites/demos/pc/app/dialog-box/webdoc/dialog-box.js:210]: 文档整改规范，<code>draggable</code>两侧需要加个空格，如：
可通过 \<code\>draggable\<\/code\> 属性设置.....
这个问题可以先不改，后续可以统一走查一下责任田组件，一起改一下~

Comment (James-9696) [reply examples/sites/demos/pc/app/dialog-box/webdoc/dialog-box.js:210]: 好哒

**Review by zzcr — DISMISSED** [review]

**Comment by kagol** [comment]
@James-9696 麻烦解决下冲突

**Comment by James-9696** [comment]
冲突已解决

**Review by kagol — DISMISSED** [review]

**Comment by kagol** [comment]
@James-9696 
以下两个 demo 看不出区别：
image

我觉得这两个 demo，应该一个是只配置 `:draggable="true"`，用来演示可拖拽这个特性；另一个配置 `:draggable="true"` 和 `:drag-outside-window="true"`，用来演示可拖拽到窗口之外。

体验链接：
https://vercel.live/open-feedback/tiny-vue-git-fork-james-9696-pr-1268-fix-opentiny.vercel.app?via=pr-comment-visit-preview-link&passThrough=1

**Comment by James-9696** [comment]
> @James-9696 以下两个 demo 看不出区别： image
> 
> 我觉得这两个 demo，应该一个是只配置 `:draggable="true"`，用来演示可拖拽这个特性；另一个配置 `:draggable="true"` 和 `:drag-outside-window="true"`，用来演示可拖拽到窗口之外。
> 
> 体验链接： https://vercel.live/open-feedback/tiny-vue-git-fork-james-9696-pr-1268-fix-opentiny.vercel.app?via=pr-comment-visit-preview-link&passThrough=1

demo已优化，可通过drag-outside-window配置看出区别

**Review by zzcr — APPROVED** [review]

**Pull request merged**